bokun.yaml

```diff
@@ -6525,44 +6945,66 @@ definitions:
         type: string
   StartTimeDto:
     properties:
-      duration:
+      id:
+        type: integer
+        format: int64
+        description: 'Specifies ID of the starting time.'
+      label:
+        type: string
+        description: 'Not working. Specifies label for this starting time.'
```

Comment:
Start times can have labels. This should be working ok.

Reply:
Agree that this is working in our GUI, however it is not working for this endpoint.

Reply:
Recommended to file a bug.